src/material/progress-spinner/_progress-spinner-theme.scss

```diff
@@ -1,27 +1,34 @@
 @use 'sass:map';
-@use '../core/mdc-helpers/mdc-helpers';
 @use '../core/theming/theming';
+@use '../core/tokens/m2/mdc/circular-progress' as tokens-mdc-circular-progress;
 @use '@material/circular-progress/circular-progress-theme' as mdc-circular-progress-theme;
 
-
 @mixin _palette-styles($config, $palette) {
   $color: theming.get-color-from-palette(map.get($config, $palette));
-  @include mdc-circular-progress-theme.theme((active-indicator-color: $color));
+  $mdc-circular-progress-color-tokens: tokens-mdc-circular-progress.get-color-tokens($config);
+
+  @include mdc-circular-progress-theme.theme(
```

Comment:
I think you can get rid of this helper mixin and just call `...theme($mdc-circular-progress-color-tokens)` for the default selector and `...theme((active-indicator-color: whatever))` for the overrides

Reply:
In the future we'll probably need to add custom tokens for `accent-active-indicator-color` and `warn-active-indicator-color`, or more likely just remove the `color="primary/accent/warn"` API